pkgs/by-name/ob/objfw/package.nix

```diff
@@ -3,18 +3,17 @@
   autogen,
   automake,
   clangStdenv,
-  fetchFromGitHub,
+  fetchfossil,
   lib,
 }:
 
 clangStdenv.mkDerivation (finalAttrs: {
   pname = "objfw";
   version = "1.2";
 
-  src = fetchFromGitHub {
-    owner = "ObjFW";
-    repo = "ObjFW";
-    rev = "refs/tags/1.2-release";
+  src = fetchfossil {
+    url = "https://objfw.nil.im/home";
+    rev = "7669b5f9d4846fd289d17f2f078c113dc8c92179ce4563256f98850660bb34d4";
```

Comment:
Maybe you can use a tag here such as `1.2-release`?

Reply:
I could well be wrong, but I don't think that works with the fossil fetcher

Reply:
turns out I was wrong, PR is updated now